spec/migrations/20151016231742_update_sample_chargeback_rate_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "spec_helper"
+require_migration
+
+describe UpdateSampleChargebackRate do
+
+  migration_context :up do
+    describe "for default Chargeback Rates" do
+      it "update description to 'Sample'" do
+        compute_rate = UpdateSampleChargebackRate::ChargebackRate.create!(:description => "Bazzillion", :rate_type => "Compute", :default => true)
```

Comment:
Please use the `migration_stub` pattern as is done in other data migration specs.